mobile/evidence/ios/EvidenceDependencies/Package.swift

```diff
@@ -6,7 +6,7 @@ import PackageDescription
 let package = Package(
     name: "EvidenceDependencies",
     platforms: [
-        .iOS(.v15), .macOS(.v10_15)
+        .iOS(.v16), .macOS(.v10_15)
```

Comment:
Why do we need this version bump?

Reply:
To add the modifier .bold() is necessary v16 or newest